package.json

```diff
@@ -9,8 +9,8 @@
     "lint": "eslint ./app",
     "start": "webpack-dev-server --config webpack.config.js --colors --env.stage=tutorial",
     "complete": "webpack-dev-server --config webpack.config.js --colors --env.stage=complete",
-    "testTutorial": "nightwatch --test nightwatch/tests/getting-started-tutorial-test.js",
-    "testComplete": "nightwatch --test nightwatch/tests/getting-started-complete-test.js"
+    "testTutorial": "node ./nightwatch/scripts/local.runner.js -c nightwatch/conf/tutorial.conf.js -e",
```

Comment:
Can you set the browsers to test with these commands -- it seems like that should work if the end command is just concatenated.

Reply:
@miyo6032 & @rylo5688 this line should have the browsers we want to test against specified.